vocabularies/scripts/update-csv.py

```diff
@@ -0,0 +1,90 @@
+# This script updates the keys in a given file with the provided values.
+#
+# Depends on Python 3
+#
+# Install dependencies:
+#   pip3 install -r requirements.txt
+#
+#
+#   Invocation:
+#   cd vocabularies/scripts
+#   python3 update-csv.py locations spaghetti.csv
+#
+#   Params:
+#    - [which]: Name of csv file to update
+#               (e.g. 'recapCustomerCodes', 'organizations').
+#    - [updateInfo] Name of csv file with values to update
+import sys
+import csv
+
+
+def csv_to_dict(file_name: str) -> dict:
+    output_dict = {}
+    row_data = {}
+
+    try:
+        with open(file_name, 'r') as f:
+            raw_csv = f.readlines()
+
+        header = raw_csv[:1][0].strip().split(',')
+        for _, row_as_string in enumerate(raw_csv[1:]):
```

Comment:
Does this reliably parse a CSV where some of the column values have commas?

Reply:
grr it does not.